proto/api.proto

```diff
@@ -800,4 +806,25 @@ message ProjectPool {
   string token1MintSymbol = 4;
   string token2MintAddress = 5;
   string token2MintSymbol = 6;
+}
+
+message GetQuotesStreamResponse {
+  int64 slot = 1;
+  string token1 = 2;
+  string token1Address = 3;
+  string token2 = 4;
+  string token2Address = 5;
+  double token1ToToken2 = 6;
```

Comment:
Shouldn't we give the paths? Or are we only doing direct routes?

Reply:
since these quotes are calculated on the fly based on the account changes that hold information relevant to reserves of a specific pool - it's always one pool... and we provide the quote for both way swaps. Should be enough for now, as a usability test. If the customers like it an need changes we'll accommodate